recipes/hydra-submitit-launcher/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "hydra-submitit-launcher" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e14c8eb46d020fac60ba25f82bcc368dc55851d2683dc95c88631ffcf15e4a34
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools
+    - wheel
+    - read-version
+    - pip
+  run:
+    - python >=3.6
+    - hydra-core >=1.1.0.dev7
+    - submitit >=1.3.3
+
+test:
+  imports:
+    - hydra_plugins.hydra_submitit_launcher
+    - hydra
+    - submitit
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/facebookincubator/submitit
+  summary: Submitit Launcher for Hydra apps
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  doc_url: https://hydra.cc/docs/plugins/submitit_launcher/
+  dev_url: https://github.com/facebookresearch/hydra/tree/main/plugins/hydra_submitit_launcher
```

Comment:
This URL assumes a branch and a sub-folder. I would leave only https://github.com/facebookresearch/hydra to prevent link rot as branches and folders are renamed/moved quicker than root URLs.